labs-ios-starter/Model/CoreData MO/Apollo/Topic/Topic/Topic+CoreDataClass.swift

```diff
@@ -63,7 +63,11 @@ public final class Topic: NSManagedObject, Codable {
         contextId = try container.decode(Int64.self, forKey: .contextId)
         timeStamp = try container.decode(String.self, forKey: .timeStamp)
 
-        section = leaderId == ProfileController.shared.authenticatedUserProfile?.id ? "Leader" : "Member"
+        guard let userId = ProfileController.shared.authenticatedUserProfile?.id else {
+            print("failed to read userId in app while decoding Topic")
+            return
+        }
+        section = leaderId == userId ? "Leader" : "Member"
```

Comment:
```suggestion
        // if leaderId is userId, 
        // section = "Leader"
        // else section = "Member"
        section = leaderId == userId ? "Leader" : "Member"
```

love how concise this  1-liner is, but this one isn't very clear at a glance. Let's try to add a comment to lines like these.